nexus/library/machines.py

```diff
@@ -195,7 +195,9 @@ def __init__(self,
                  machine      = None,
                  account      = None,
                  queue        = None,
-                 bundled_jobs = None,
+		 group_list   = None,
```

Comment:
Indentation here still looks funny - mixed spaces and tabs?

Reply:
Now  I see it's in the argument list where indentation is not significant.    It would be nice if the indentation matches the surrounding code, though.